### PR TITLE
Improvement/leo.pereyra/check in

### PR DESCRIPTION
**Pull request by LeonardoPC23**
Add checkin checkout

### CODE REVIEW

File: pdv/src/main/java/com/chuys/gshp/pdv/data/repository/KpiDataRepository.kt [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+    private fun getSite(): Single<Resource<KpiData>> {
+        return Single.create {
+            var kpiData: KpiData? = null
+            referenceDb.addValueEventListener(object : ValueEventListener {
+                override fun onDataChange(dataSnapshot: DataSnapshot) {
+                    try {
+                        kpiData = dataSnapshot.getValue(KpiData::class.java)
+                    } catch (e: Exception) {
+
```
Comment (antoniunix) [inline pdv/src/main/java/com/chuys/gshp/pdv/data/repository/KpiDataRepository.kt]: please put catch

File: pdv/src/main/java/com/chuys/gshp/pdv/data/repository/PdvDataRepository.kt [hunk 2]
```diff
@@ -43,6 +43,8 @@ class PdvDataRepository : PdvRepository {
                         }
                         moduleList.add(pdv!!)
                     }
+
```
Comment (antoniunix) [inline pdv/src/main/java/com/chuys/gshp/pdv/data/repository/PdvDataRepository.kt:46]: delete extra space

File: pdv/src/main/java/com/chuys/gshp/pdv/data/repository/ReportReportEntity.kt [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    private val tableName = "report_report"
+
+
+
+    fun writeReportReport(value: ReportReportModel) {
+        val db = sqliteHelper.writableDatabase
+
+        try {
+            var insStmnt = db.compileStatement(
+                "" +
```
Comment (antoniunix) [inline pdv/src/main/java/com/chuys/gshp/pdv/data/repository/ReportReportEntity.kt:17]: is necesary?

File: pdv/src/main/java/com/chuys/gshp/pdv/view/AddPdv.kt [hunk 4]
```diff
@@ -11,52 +19,88 @@ import com.chuys.gshp.shared.data.job.UIThread
...
+        if (requestCode == IntConstants.LOCATION_ACTIVITY_REQUEST_CODE) {
+            finish()
+           intent = Intent(this, AddPdv::class.java)
+            startActivity(intent)
+        }
+    }
+
+    @SuppressLint("MissingSuperCall")
+    public override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
+        if (requestCode == IntConstants.LOCATION_ACTIVITY_REQUEST_CODE) {
```
Comment (antoniunix) [inline pdv/src/main/java/com/chuys/gshp/pdv/view/AddPdv.kt:77]: this only generate technical debt, join the "if"

File: pdv/src/main/java/com/chuys/gshp/pdv/data/repository/ReportReportEntity.kt (hunk 3 above)
Comment (LeonardoPC23) [reply pdv/src/main/java/com/chuys/gshp/pdv/data/repository/ReportReportEntity.kt:17]: no, it was a temporary change, due to the lack of knowledge that a report was already generated

File: pdv/src/main/java/com/chuys/gshp/pdv/data/repository/KpiDataRepository.kt (hunk 1 above)
Comment (LeonardoPC23) [reply pdv/src/main/java/com/chuys/gshp/pdv/data/repository/KpiDataRepository.kt]: ok

**Pull request merged**